soc/renesas/ra/ra4m1/Kconfig

```diff
@@ -9,3 +9,26 @@ config SOC_SERIES_RA4M1
 	select DYNAMIC_INTERRUPTS
 	select TIMER_READS_ITS_FREQUENCY_AT_RUNTIME
 	select XIP
+
+config SOC_OPTION_SETTING_MEMORY
+	bool "Option Setting Memory"
+	default y
+
+config SOC_OPTION_SETTING_LVDAS_DISABLE
+	bool "Disable voltage monitor 0 after reset"
+	default y
+	depends on SOC_OPTION_SETTING_MEMORY
+
+config SOC_OPTION_SETTING_HOCO_DISABLE
```

Comment:
Whether the hoco is valid or invalid, I don't think there is any problem with judging it by the following code.

````
DT_NODE_HAS_STATUS(DT_PATH(clocks, hoco), okay)
````

If you are considering a case where the not enabled the hoco at reset but is enabled during startup processing, add a property to `dts/bindings/clock/renesas,ra-clock-generation-circuit.yaml`. I think it's also good. (This might be overthinking.)

Reply:
This is same as `SOC_OPTION_SETTING_HOCO_FREQ`.

https://github.com/zephyrproject-rtos/zephyr/pull/71017/files#diff-7cd037a8cbd8c3e02942d0cbda9cf90aa08fbab8ab55dcf57b46e12f660662beR122

This line can rewrite 

```
		.HOCOEN = !DT_NODE_HAS_STATUS(DT_PATH(clocks, hoco), okay),
```

And we can remove this Kconfig option.

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/71017/files#diff-7cd037a8cbd8c3e02942d0cbda9cf90aa08fbab8ab55dcf57b46e12f660662beR10-R20

It seems to be a good. We can remove the config.

```
config SOC_OPTION_SETTING_HOCO_DISABLE
```